### PR TITLE
Fix ambiguous variable bindings in Exercise 10.12, 10.13

### PR DESCRIPTION
**Pull request by koba-e964**
Fix https://github.com/HoTT/book/issues/910.

### CODE REVIEW

File: errata.tex [hunk 1]
```diff
@@ -769,6 +769,9 @@
   \cref{thm:VisCST}
   & 706-ged2c765
   & In the proof of the function set axiom, ``the types of elements $[u] \mono V$ and $[u] \mono V$'' should be ``the types of members $[u] \mono V$ and $[v] \mono V$.''\\
+  \cref{ex:strong-collection}
+  & % merge 0c4f868
+  & Extra parentheses are needed to make the formula unambiguous. \\
```
Comment (mikeshulman) [inline errata.tex]: Might as well be explicit here about where the parentheses go?

**Comment by koba-e964** [comment]
I see. I added some information.

File: errata.tex [hunk 2]
```diff
@@ -769,6 +769,12 @@
   \cref{thm:VisCST}
   & 706-ged2c765
   & In the proof of the function set axiom, ``the types of elements $[u] \mono V$ and $[u] \mono V$'' should be ``the types of members $[u] \mono V$ and $[v] \mono V$.''\\
+  \cref{ex:strong-collection}
+  & % merge 0c4f868
+  & Extra parentheses around $\fall{x\in v}\exis{y} R(x,y)$ are needed to make the formula unambiguous. \\
+  \cref{ex:choice-cumulative-hierarchy-choice}
+  & % merge c395527
+  & Extra parentheses around $\fall{y\in x}\exis{z\in V} z\in y$ are needed to make the formula unambiguous. In addition, the big parentheses after $\fall{x\in V}$ are added for clarity.\\
```
Comment (mikeshulman) [inline errata.tex]: Sorry to be such a nitpicker, but I wouldn't mention the "for clarity" parentheses.  But if you disagree, feel free to say so.  My rule of thumb is to think about who is reading the errata: it's not people who are reading the correct version, it's people who have the incorrect version and need to know what is wrong with it.  Ideally, we should tell them exactly what's mathematically wrong and how to fix it (though space doesn't always allow us to be fully detailed); but I don't think they need to know about cosmetic changes that we also made in the updated version.

**Comment by koba-e964** [comment]
I've thought that errata should enumerate the difference in changes, but now I understand and agree. I fixed it.

**Pull request merged**